### PR TITLE
[idea] #182 fixed no class def error for HTTP methods in idea 13

### LINKED ISSUE #182: Idea plugin throwing exception for Intellij idea version 13

### PR DESCRIPTION
**Pull request by prudhvije**
fixes #182

### CODE REVIEW

**Comment by prudhvije** [comment]
Results in idea 13

image

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/util/ESUtils.java [hunk 1]
```diff
@@ -123,28 +121,27 @@ protected final JsonArray getJsonElements(final String esResultJson) {
...
 
             BufferedReader bufferedReader = new BufferedReader(
-                    new InputStreamReader((response.getEntity().getContent()), UTF_8));
+                    new InputStreamReader(getMethod.getResponseBodyAsStream(), UTF_8));
             String output;
             while ((output = bufferedReader.readLine()) != null) {
                 stringBuilder.append(output);
             }
             bufferedReader.close();
-            httpClient.getConnectionManager().shutdown();
```
Comment (ScrapCodes) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/util/ESUtils.java]: We should close httpClient ?

**Comment by ScrapCodes** [comment]
I have just updated the travis build to run against Intellij 13.1.6 and wonder how it passes.

**Comment by prudhvije** [comment]
Plugin supporting idea builds from 135 only. This happens in versions prior to 135.

**Pull request closed without merging**